app/scripts/Xprize-UserInterfacesRetargeting.xml

```diff
@@ -112,20 +112,20 @@
       <port timeout="11.0">/iFeelSuit/WearableActuatorsCommand/input:i</port>
     </dependencies>
   </module>
-  
+
   <!-- Connections -->
-  <connection>
-    <from>/SRanipalModule/emotions:o</from>
-    <to>/icub/face/emotions/in</to>
-    <protocol>fast_tcp</protocol>
-  </connection>
-  
-  <connection>
-    <from>/SRanipalModule/VR/rpc:o</from>
-    <to>/joypadDevice/Oculus/rpc</to>
-    <protocol>fast_tcp</protocol>
-  </connection>
-  
+  <!-- <connection> -->
+  <!--   <from>/SRanipalModule/emotions:o</from> -->
+  <!--   <to>/icub/face/emotions/in</to> -->
+  <!--   <protocol>fast_tcp</protocol> -->
+  <!-- </connection> -->
+
+  <!-- <connection> -->
+  <!--   <from>/SRanipalModule/VR/rpc:o</from> -->
+  <!--   <to>/headset/rpc</to> -->
+  <!--   <protocol>fast_tcp</protocol> -->
+  <!-- </connection> -->
+
```

Comment:
Probably we can just remove this

Reply:
Done in [769e045](https://github.com/robotology/walking-teleoperation/pull/108/commits/769e0458a0bb7199cc50ab7ba4299bfdf227ef2f)